translate_demo.py

```diff
@@ -282,7 +283,10 @@ async def main(mode = 'demo'):
 
 	print(' -- Loading models')
 	os.makedirs('result', exist_ok=True)
-	text_render.prepare_renderer()
+    if args.font_filename:
+        text_render.prepare_renderer([args.font_filename])
+    else:
+        text_render.prepare_renderer()

```

Comment:
1. Use tabs not space or you get an indentationerror
2. Inlined would look better

Reply:
changed space to same tabs
hard to me to understand inline way

Reply:
I meant `text_render.prepare_renderer(font_path.split(',') if args.font_path else '')` but that probably wouldnt work in this case as it would overwrite the default value.